doc/how-to/recover.md

```diff
@@ -0,0 +1,63 @@
+# MicroCloud Cluster Recovery
+
+Each MicroCloud service uses the [Dqlite](https://dqlite.io/) distributed
+database for highly-available storage. While the cluster recovery process is
+similar for each service, this document only covers cluster recovery for the
+`microcloudd` daemon. For cluster recovery procedures for LXD, MicroCeph and
+MicroOVN, see:
+
+- [LXD Cluster Recovery](https://documentation.ubuntu.com/lxd/en/latest/howto/cluster_recover/)
+- [MicroOVN Launchpad Bug](https://bugs.launchpad.net/microovn/+bug/2072377)
+- [MicroCeph Issue](https://github.com/canonical/microceph/issues/380)
+
+If a MicroCloud cluster loses more than a quorum of its database members, then
+database operations will no longer be possible on the entire cluster. If the
+loss of quorum is temporary (e.g. some members temporarily lose power), database
+operations will be restored when the down members come back online.
+
+This document describes how to recover database access if the down members have
+been lost without the possibility of recovery (e.g. disk failure).
+
+## Recovery Procedure
+
+1. In order to perform cluster recovery, **all cluster members must be shut down**:
+```
+sudo snap stop microcloud
+```
+
+1. Once all cluster members are shut down, determine which Dqlite database is
+most up to date. Look for files in `/var/snap/microcloud/common/state/database` whose
+filenames are two numbers separated by a dash (i.e.
+`0000000000056436-0000000000056501`). The largest second number in the directory
+is the end-index of the most recently closed segment (56501 in the example).
+Perform the next step on the cluster member with the highest end-index.
+
+2. Use the following command **on one cluster member** to reconfigure the Dqlite
+roles for each member:
+```
+sudo microcloud cluster edit
+```
```

Comment:
````suggestion
1. On the cluster member with the most up-to-date database, reconfigure the Dqlite roles for each member with the following command:
   ```
   sudo microcloud cluster edit
   ```
````

All steps should use `1.` - with the correct indentation this should work. :wink:
The advantage is that you can later insert a step and don't need to think about updating the numbers.